ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -792,6 +1059,10 @@ private[platform] object JdbcLedgerDao {
       validatePartyAllocation: Boolean = false,
       jdbcAsyncCommitMode: DbType.AsyncCommitMode = DbType.SynchronousCommit,
       enricher: Option[ValueEnricher],
+      sequentialIndexer: SequentialIndexer = NoopSequentialIndexer,
+      participantId: com.daml.ledger.participant.state.v1.ParticipantId = "".asInstanceOf[
```

Comment:
This can't be right to assign an empty participant. sandbox-classic takes a participant-id as a command line argument.

Reply:
the sequentialIndexer and the participantId is only used for the sandbox-classic integration, so I thought is it a good idea to limit the dependancy this way.

Reply:
let's do that to prevent confusion in future